docs/dev/index.rst

```diff
@@ -396,3 +396,11 @@ Security
    :maxdepth: 1
 
    security
+
+
+microTVM
+---------
+.. toctree::
```

Comment:
as a general principle we don't check in figures. Please checkin figures to another repo and then reference from the docs e.g. https://raw.githubusercontent.com/tlc-pack/web-data/main/images/design/tvm_static_overview.svg

Reply:
why's that? these are text figures and making people upload to an external repository sort of discourages figures.

Reply:
The main reason is that many figures are binaries, updating figures would corresponds to the diff of binaries. Adding binaries as part of the repo would make the git history itself inevitably large.

Reply:
https://github.com/tlc-pack/web-data/pull/4

Reply:
Looks great and LGTM. Thank you @areusch .